src/frontend/src/views/Upload/components/Metadata/components/ImportFile/components/Alerts/warnings.tsx

```diff
@@ -82,6 +83,52 @@ export function WarningExtraneousEntry({
   );
 }
 
+/**
+ * WARNING_CODE.EXTRANEOUS_ENTRY (SAMPLE EDIT)
+ * (mostly similar to the above implementation, wording is different enough to have it's own component)
+ */
+function MessageExtraneousEntrySampleEdit({
+  extraneousSampleIds,
+}: PropsExtraneousEntry) {
+  const tablePreamble =
+    "The following IDs in your file’s “Current Private ID” column did not match any selected " +
+    "samples, and weren’t imported. Please double check and correct any errors. ";
+  const columnHeaders = [SAMPLE_EDIT_METADATA_KEYS_TO_HEADERS.currentPrivateID];
+  const rows = extraneousSampleIds.map((sampleId) => [sampleId]);
+  return (
+    <ProblemTable
+      tablePreamble={tablePreamble}
+      columnHeaders={columnHeaders}
+      rows={rows}
+    />
+  );
+}
+
+export function WarningExtraneousEntrySampleEdit({
+  extraneousSampleIds,
+}: PropsExtraneousEntry): JSX.Element {
+  const count = extraneousSampleIds.length;
+  // "X Samples in metadata file were not used."
+  const title = `${count} ${pluralize(
```

Comment:
I think this works out to `5 Samples in metadata files were couldn't be matched and weren't imported.`. Is that intentional?

It seems like maybe the pluralization of "was" should be moved to just before imported